packages/vitest/src/runtime/worker.ts

```diff
@@ -100,13 +68,21 @@ export async function execute(method: 'run' | 'collect', ctx: ContextRPC, worker
     await Promise.all(cleanups.map(fn => fn()))
 
     await rpcDone().catch(() => {})
-    environmentLoader?.close()
+    await environmentLoader?.close()
+    rpc.$close()
```

Comment:
We should pass `off` method from Worker inits, so that `birpc` removes its listener here automatically.

- https://github.com/antfu-collective/birpc/blob/559f621162137194b9c0d4456b90a7f15ea7a135/src/index.ts#L299